### PR TITLE
1652 local docker db stats

### PR DESCRIPTION
**Pull request by Red-HAP**
## Description

Local DB container initialization changes to enable statement statistics. This change will operate in conjunction with the other changes for large data generation.

This was accomplished without needing to build a new image.

### Setup

`make docker-reinitdb`
Then use psql to connect to the local db

### Copleted Local Tests

- [ ] `show shared_preload_libraries;` should show `pg_stat_statements`
- [ ] `\dx` should show `pg_stat_statements`
- [ ] `select * from pg_stat_statements;` should return records if executed after the first two tests

### CODE REVIEW

File: pg_init/99_postgresql_conf.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+					#   debug1
+					#   info
+					#   notice
+					#   warning
+					#   error
+					#   log
+					#   fatal
+					#   panic (effectively off)
+
+log_min_duration_statement = 5000	# -1 is disabled, 0 logs all statements
```
Comment (adberglund) [inline pg_init/99_postgresql_conf.sh]:
```suggestion
log_min_duration_statement = 2000	# -1 is disabled, 0 logs all statements
```

Comment (adberglund) [reply pg_init/99_postgresql_conf.sh]: This again is just to stay consistent with what the platform considers latency downtime today. I know they're tracking request/response time and that's certainly not all DB time, but this at least gets us close to that metric.

**Comment by kholdaway** [comment]
Nice!

File: pg_init/99_postgresql_conf.sh (hunk 1 above)
Comment (Red-HAP) [reply pg_init/99_postgresql_conf.sh]: This has been changed to 2000 ms

Comment (Red-HAP) [reply pg_init/99_postgresql_conf.sh]: Yep. Forgot to make that change initally.

**Pull request merged**